src/pages/drivers/[driver].tsx

```diff
@@ -70,7 +70,31 @@ const individualDriverSchema = z.object({
 })
 
 function driverWins() {
+  // const router = useRouter()
+  // const { driver } = router.query
+
+  // const [driverQuery, resultsQuery] = useQueries({
+  //   queries: [
+  //     {
+  //       queryKey: ['individual_driver', driver],
+  //       queryFn: () => getDriver(String(driver)),
+  //     },
+  //     {
+  //       queryKey: ['results', driver],
+  //       queryFn: () => getResults(String(driver)),
+  //     },
+  //   ],
+  // })
+  // var wins = 0
+  // for (let i = 0; i < resultsQuery.data.length; i++)
+  // {
+  //   if (resultsQuery.data[i].Results[0].position === "1")
+  //   {
+  //     wins += 1
+  //   }
+  // }
   return
+  // return wins
```

Comment:
This is commented out, but for reference: you're using a [react-query hook `useQueries`](https://tanstack.com/query/v4/docs/react/reference/useQueries) and a [NextJS hook `useRouter`](https://nextjs.org/docs/api-reference/next/router) inside a normal function (i.e. not a React component), so it won't work. A React component using hooks either returns [JSX](https://facebook.github.io/jsx/) or the return value of a [custom hook](https://reactjs.org/docs/hooks-custom.html)

Reply:
Done